dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/newsession/NewSessionPresenter.java

```diff
@@ -104,10 +145,10 @@ private void initSessionList() {
                 model.setSessionName(newSelection.getName());
                 model.setEndpoint(newSelection.getEndpoint());
                 model.setAccessKey(newSelection.getCredentials().getAccessId());
-                model.setPortno(newSelection.getPortNo());
                 model.setSecretKey(newSelection.getCredentials().getSecretKey());
+                model.setPortno(newSelection.getPortNo());
                 model.setProxyServer(newSelection.getProxyServer());
-                if (newSelection.getDefaultSession() == null) {
+                if (newSelection.getDefaultSession() == null || newSelection.getDefaultSession() == null) {
```

Comment:
oops, that should be reverted.  Changes to the "Default Session" setting weren't being persisted, but I fixed that elsewhere.  Thanks for the catch.